src/store/modules/shipment/actions.ts

```diff
@@ -18,9 +18,8 @@ const actions: ActionTree<ShipmentState, RootState> = {
         "statusId": "PURCH_SHIP_SHIPPED"
       })
       if (resp.status === 200 && resp.data.count> 0 && !hasError(resp)) {
-        let shipments = resp.data.docs;
-
-        if (payload.viewIndex && payload.viewIndex > 0) shipments = state.shipments.list.concat(shipments)
```

Comment:
Why we have removed this concat condition?

Reply:
Because this if condition is breaking execution of our code because there is no need of cancat() method it automatically adds upcoming shipments in state. There is an additional point is this concat method only works for string not for objects.